relayer/relayer/src/contracts/azero.rs

```diff
@@ -140,7 +173,7 @@ impl MostInstance {
             .await??)
     }
 
-    pub async fn needs_signature(
+    pub async fn _needs_signature(
```

Comment:
checking this could still save guardian some gas if request collected enough signatures before he committed his

Reply:
We currently have no way of calling this for a finalized block, only for latest so using this is unsafe unfortunately.